casbin/enforcer.h

```diff
@@ -66,6 +66,18 @@ class Enforcer : public IEnforcer {
          * @param policy_file the path of the policy file.
          */
         Enforcer(const std::string& model_path, const std::string& policy_file);
+
+        // Why we need 2 declarations? Asked a question:
+        // FIXME: https://github.com/casbin/casbin-cpp/issues/201
+        /**
+         * Enforcer initializes an enforcer with a model file and a policy string stream.
+         *
+         * @param model_path the path of the model file.
+         * @param policy the policy string stream.
+         */
+        Enforcer(const std::string& model_path, std::stringstream && policy);

```

Comment:
We need the same declaration in one more place: in casbin_enforcer.h. don't have a clue why. asked question: https://github.com/casbin/casbin-cpp/issues/201

Reply:
The other file include/casbin/casbin_enforcer.h is for python binding:
`This is the main file for python bindings workflow`

Reply:
oh, yea, the problem is if I do not change both files - it does fails on compilation... So required both changes